src/meta/src/manager/cluster.rs

```diff
@@ -86,6 +87,39 @@ where
         self.core.read().await
     }
 
+    pub async fn start_worker_num_monitor(
```

Comment:
Would it be better if we only update the metrics on `add_worker_node`, `update_worker_node` and `delete_worker_node`?

Reply:
That's the original approach

Reply:
I meant setting the absolute metric value directly on those function calls instead of calling `inc` and `dec`. I don't have a strong opinion on this and the current implementation also works.

Reply:
Got it. Either LGMT 😜